metals/src/main/scala/scala/meta/internal/metals/codeactions/CreateNewSymbol.scala

```diff
@@ -35,7 +35,7 @@ class CreateNewSymbol() extends CodeAction {
 
     val codeActions = params.getContext().getDiagnostics().asScala.collect {
       case d @ ScalacDiagnostic.SymbolNotFound(name)
-          if d.getRange().encloses(params.getRange()) =>
+          if params.getRange().overlapsWith(d.getRange()) =>
```

Comment:
It funny because I remember reading your comment about this on the first pr, and it totally didn't even register that this wasn't going to work in coc-metals 😆 . Thanks for fixing this!

Reply:
O, and I did test this locally, and it works great :)

Reply:
Cool, thanks for confirming!